suite-common/suite-utils/src/pollingController.ts

```diff
@@ -0,0 +1,39 @@
+type PollingCallback = () => Promise<void> | void;
+
+export const createPollingController = () => {
+    let intervalId: ReturnType<typeof setInterval> | null = null;
+    let running = false;
+
+    const start = (cb: PollingCallback, intervalMs: number) => {
```

Comment:
don't use unnecessary abbreviations please :pray: 
```suggestion
    const start = (callback: PollingCallback, intervalMs: number) => {
```